cmake/modules/RootCPack.cmake

```diff
@@ -72,10 +72,12 @@ if(MSVC)
     math(EXPR VS_VERSION "${VC_MAJOR} - 6")
   elseif(MSVC_VERSION LESS 1910)
     math(EXPR VS_VERSION "${VC_MAJOR} - 5")
-  elseif(MSVC_VERSION LESS 1919)
+  elseif(MSVC_VERSION LESS 1920)
     math(EXPR VS_VERSION "${VC_MAJOR} - 4")
-  elseif(MSVC_VERSION LESS 1925)
+  elseif(MSVC_VERSION LESS 1930)
     math(EXPR VS_VERSION "${VC_MAJOR} - 3")
+  elseif(MSVC_VERSION LESS 1940)
+    math(EXPR VS_VERSION "${VC_MAJOR} - 2")
```

Comment:
Can we add an error case so we detect this issue next time?

Reply:
Sure!